packages/server-wallet/src/models/channel.ts

```diff
@@ -154,6 +157,14 @@ export class Channel extends Model implements RequiredColumns {
       .first();
   }
 
+  static async setInitialSupport(
```

Comment:
Can you add the `CHALLENGING_V0` tag here, so we remember to remove it along with the below code?

I think that would be sufficient, since removing this would trigger a cascade of compile errors that leads to the remaining cleanup code.

Reply:
Done in 26da120